_data/tutorials.yaml

```diff
@@ -214,3 +214,14 @@ flatten-nested-data:
     ksql: enabled
     kstreams: disabled
     kafka: disabled
+
+aggregating-average:
+  title: "How to compute average?"
+  meta-description: "How to compute average?"
+  slug: "/aggregating-average"
+  problem: "Kafka Streams DSL aggregation natively support so-called incremental\" or \"cumulative\" aggregation functions like count, sum, min, max. However, average function can be composed of distributive functions, namely count and sum. In this tutorial we will write a program."
```

Comment:
```suggestion
  problem: "Kafka Streams natively supports \"incremental\" aggregation functions, in which the aggregation result is updated based on the values captured by each window. Incremental functions include count, sum, min, and max. An average aggregation cannot be computed incrementally. However, as this tutorial shows, it can be implemented by composing incremental functions, namely count and sum."
```